model/common/tests/grid_tests/test_vertical.py

```diff
@@ -8,11 +8,11 @@
 import math
 
 import gt4py.next as gtx
-import numpy as np
 import pytest
 
 from icon4py.model.common import dimension as dims
 from icon4py.model.common.grid import vertical as v_grid
+from icon4py.model.common.settings import xp
```

Comment:
Outcome of a discussion in last cycle is that the test ground truth should always be on CPU. So you can use the numpy import here.
Then in the test you compute gt4py stuff on varying backends which is given by the parametrizable `backend` fixture, the ground truth to compare to (that can be serialbox which reads in (-> CPU) or `numpy` on CPU. Then for the comparison itself you need to do a `x.asnumpy()` and do the comparison on CPU.